lib/device/library/linux.rb

```diff
@@ -2,22 +2,76 @@
 #
 # Copyright 2013 whiteleaf. All rights reserved.
 #
+# rubocop:disable Lint/AssignmentInCondition
+# rubocop:disable UtilityFunction
+
 require "etc"
 
-module Device::Library
-  module Linux
-    username = Etc.getpwuid(Process.euid).name
-    @@mount_roots = %w(/media /mnt)
-    @@mount_roots << "/run/media/" + username # udisks2 による自動マウント(デフォルトの場所)
-    @@mount_roots << "/media/" + username # udisks2 による自動マウント(Ubuntuの場合)
-    def get_device_root_dir(volume_name)
-      @@mount_roots.each do |mount_root|
-        path = File.join(mount_root, volume_name)
-        if File.directory?(path)
-          return path
+class Device
+  module Library
+    module Linux
+      username = Etc.getpwuid(Process.euid).name
+      @@mount_roots = %w(/media /mnt)
+      @@mount_roots << "/run/media/" + username # udisks2 による自動マウント(デフォルトの場所)
+      @@mount_roots << "/media/" + username # udisks2 による自動マウント(Ubuntuの場合)
+      def get_device_root_dir(volume_name)
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)

<!-- Comment by [SideCI](https://sideci.com) -->